script.js

```diff
@@ -1,12 +1,88 @@
-//You can edit ALL of the code here
+const inputSearch = document.getElementById("search-input");
+const selectTag = document.getElementById("episodes");
+const resetBtn = document.querySelector(".btn-reset");
+const rootElem = document.getElementById("root");
+const allEpisodes = getAllEpisodes();
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
+  selectEpisode(allEpisodes);
 }
 
+// Loads episodes cards
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
```

Comment:
Good work! everything is well organised,, I liked how you named all your variables, and you completed all levels.